server/server_test.go

```diff
@@ -86,6 +87,13 @@ func RunServer(opts *Options) *Server {
 		s.ConfigureLogger()
 	}
 
+	if ll := os.Getenv("NATS_LOGGING"); ll != "" {
+		log := srvlog.NewTestLogger(fmt.Sprintf("[%-5s] | ", opts.ServerName), true)
```

Comment:
Not sure of the need of the `%-5s`. It looks weird for small names like `A` or `B`, but I get that if you have small names (but less than 5 characters), at least the output is "aligned".
```
[B    ] | 2024/03/11 09:30:14.319038 [INF] Starting nats-server
```
I would have gone with a simple `fmt.Sprintf("[%s] ",`.

But I would recommend replacing `opts.ServerName` with `s` otherwise, when the server name is not set in the options, you would get something like:
```
=== RUN   TestGatewayBasic
[     ] | 2024/03/11 09:26:47.580013 [INF] Starting nats-server
[     ] | 2024/03/11 09:26:47.580029 [INF]   Version:  2.11.0-dev
[     ] | 2024/03/11 09:26:47.580037 [INF]   Git:      [not set]
...
```
When the name is not set, `s` (which is Server.String()) would return the server ID, which will at least allow differentiating between servers:
```
=== RUN   TestGatewayBasic
[NBZZHUIG3KYM6D434GN3PDVSKDM3UC6NGJF7DG4S4NKVN6AGSD2J3HHI] | 2024/03/11 09:29:13.391026 [INF] Starting nats-server
[NBZZHUIG3KYM6D434GN3PDVSKDM3UC6NGJF7DG4S4NKVN6AGSD2J3HHI] | 2024/03/11 09:29:13.391041 [INF]   Version:  2.11.0-dev
[NBZZHUIG3KYM6D434GN3PDVSKDM3UC6NGJF7DG4S4NKVN6AGSD2J3HHI] | 2024/03/11 09:29:13.391052 [INF]   Git:      [not set]
```

Reply:
Have made these changes!